src/domain/customers/CustomerListComponent.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+
+import Table from '../../common/table/Table';
+import CustomerDetails from './CustomerDetailsComponent';
+
+export interface CustomerData {
+  comment?: string;
+  contactMethod?: string;
+  email?: string;
+  firstName?: string;
+  id?: string;
+  image?: string;
+  invoicingType?: string;
+  language?: string;
+  lastName?: string;
+  nickname?: string;
+  phone?: string;
+}
+
+interface TableData {
+  goToDetails: string;
+  group: string;
+  invoice: string;
+  name: string;
+  queue: string;
+  startDate: string;
+  thing: string;
+}
+
+interface Props {
+  data: CustomerData | null | any;
```

Comment:
Also here, better to avoid `any`, since it just suppresses TypeScript errors.

Reply:
same as above